### PR TITLE
Improve detection of Chrome channel variants

### PR DESCRIPTION
**Pull request by pimterry**
Working towards a fix for #55, based on the comments there. This implements the suggested change, and improves the detection in general, especially for beta & dev channels.

Output of `node cli.js --debug` with lots of Chromes installed on a Windows 10 VM:

```
$ node cli.js --debug
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Google\Update /v LastInstallerSuccessLaunchCmdLine +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ HKCU\Software\Google\Update /v LastInstallerSuccessLaunchCmdLine +4ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe +2ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ HKCU\Software\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe +1ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Clients\StartMenuInternet\Google Chrome\shell\open\command +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Clients\StartMenuInternet\Google Chrome\shell\open\command +0ms
  win-detect-browsers Found chrome (32):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
  win-detect-browsers   @ Software\Wow6432Node\Google\Update\ClientState\{401C381F-E0DE-4B85-8BD8-3F3F14FBDA57}\LastInstallerSuccessLaunchCmdLine +1ms
  win-detect-browsers Found chrome (32):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ Software\Wow6432Node\Google\Update\ClientState\{8237E44A-0054-442C-B6B6-EA0509993955}\LastInstallerSuccessLaunchCmdLine +1ms
  win-detect-browsers Found chrome (stable) (64):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ Software\Wow6432Node\Google\Update\ClientState\{8A69D345-D564-463C-AFF1-A69D9E530F96}\LastInstallerSuccessLaunchCmdLine +0ms
  win-detect-browsers Found chrome (canary) (64):
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ Software\Google\Update\ClientState\{4EA16AC7-FD5A-47C3-875B-DBF4A2008C20}\LastInstallerSuccessLaunchCmdLine +0ms
  win-detect-browsers firefox: found key "70.0.1 (x86 en-US)" in HKLM\Software\Wow6432Node\Mozilla\Mozilla Firefox /v CurrentVersion +1ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files\Internet Explorer\iexplore.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Clients\StartMenuInternet\iexplore.exe\shell\open\command +0ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files\Internet Explorer\iexplore.exe
  win-detect-browsers   @ HKLM\Software\Clients\StartMenuInternet\iexplore.exe\shell\open\command +1ms
  win-detect-browsers Found chrome (canary):
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeSSHTM.BC355NTUYZBE365DOMSF45YR2M\shell\open\command +1ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeHTML\shell\open\command +0ms
  win-detect-browsers Found chrome (dev):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeDHTML\shell\open\command +0ms
  win-detect-browsers Found chrome (beta):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeBHTML\shell\open\command +0ms
  win-detect-browsers Found firefox:
  win-detect-browsers   - C:\Program Files (x86)\Mozilla Firefox\firefox.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Mozilla\Mozilla Firefox\70.0.1 (x86 en-US)\Main /v PathToExe +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ default location +2ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ default location +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ default location +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
  win-detect-browsers   @ default location +0ms
  win-detect-browsers Found firefox:
  win-detect-browsers   - C:\Program Files (x86)\Mozilla Firefox\firefox.exe
  win-detect-browsers   @ default location +2ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files (x86)\Internet Explorer\iexplore.exe
  win-detect-browsers   @ default location +0ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files\Internet Explorer\iexplore.exe
  win-detect-browsers   @ default location +1ms
CHROME 79 BETA 64-bit
├── Path:    C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
├── Version: 79.0.3945.36
├── GUID:    8237E44A-0054-442C-B6B6-EA0509993955
├─┬ Info
│ ├── FileVersion:      79.0.3945.36
│ ├── CompanyName:      Google LLC
│ ├── FileDescription:  Google Chrome
│ ├── InternalName:     chrome_exe
│ ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
│ ├── OriginalFilename: chrome.exe
│ ├── ProductName:      Google Chrome
│ ├── ProductVersion:   79.0.3945.36
│ ├── CompanyShortName: Google
│ ├── ProductShortName: Chrome
│ ├── LastChange:       3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}
│ └── Official Build:   1
└─┬ Uninstall
  ├── path: C:\Program Files (x86)\Google\Chrome Beta\Application\79.0.3945.36\Installer\setup.exe
  └─┬ arguments
    ├── --uninstall
    ├── --chrome-beta
    ├── --system-level
    └── --verbose-logging

CHROME 80 CANARY 64-bit
├── Path:    C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
├── Version: 80.0.3968.0
├── GUID:    4EA16AC7-FD5A-47C3-875B-DBF4A2008C20
├─┬ Info
│ ├── FileVersion:      80.0.3968.0
│ ├── CompanyName:      Google LLC
│ ├── FileDescription:  Google Chrome
│ ├── InternalName:     chrome_exe
│ ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
│ ├── OriginalFilename: chrome.exe
│ ├── ProductName:      Google Chrome
│ ├── ProductVersion:   80.0.3968.0
│ ├── CompanyShortName: Google
│ ├── ProductShortName: Chrome
│ ├── LastChange:       942a57022a321872a10830a40120904d79bd0536-refs/branch-heads/3968@{#1}
│ └── Official Build:   1
└─┬ Uninstall
  ├── path: C:\Users\User\AppData\Local\Google\Chrome SxS\Application\80.0.3968.0\Installer\setup.exe
  └─┬ arguments
    ├── --uninstall
    └── --chrome-sxs

CHROME 78 STABLE 64-bit
├── Path:    C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
├── Version: 78.0.3904.97
├── GUID:    8A69D345-D564-463C-AFF1-A69D9E530F96
├─┬ Info
│ ├── FileVersion:      78.0.3904.97
│ ├── CompanyName:      Google LLC
│ ├── FileDescription:  Google Chrome
│ ├── InternalName:     chrome_exe
│ ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
│ ├── OriginalFilename: chrome.exe
│ ├── ProductName:      Google Chrome
│ ├── ProductVersion:   78.0.3904.97
│ ├── CompanyShortName: Google
│ ├── ProductShortName: Chrome
│ ├── LastChange:       021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
│ └── Official Build:   1
└─┬ Uninstall
  ├── path: C:\Program Files (x86)\Google\Chrome\Application\78.0.3904.97\Installer\setup.exe
  └─┬ arguments
    ├── --uninstall
    ├── --system-level
    └── --verbose-logging

CHROME 80 DEV 64-bit
├── Path:    C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
├── Version: 80.0.3964.0
├── GUID:    401C381F-E0DE-4B85-8BD8-3F3F14FBDA57
├─┬ Uninstall
│ ├── path: C:\Program Files (x86)\Google\Chrome Dev\Application\80.0.3964.0\Installer\setup.exe
│ └─┬ arguments
│   ├── --uninstall
│   ├── --chrome-dev
│   ├── --system-level
│   └── --verbose-logging
└─┬ Info
  ├── FileVersion:      80.0.3964.0
  ├── CompanyName:      Google LLC
  ├── FileDescription:  Google Chrome
  ├── InternalName:     chrome_exe
  ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
  ├── OriginalFilename: chrome.exe
  ├── ProductName:      Google Chrome
  ├── ProductVersion:   80.0.3964.0
  ├── CompanyShortName: Google
  ├── ProductShortName: Chrome
  ├── LastChange:       75f9d4c9f31c6fa9302434e99ced85dc5d8e2069-refs/branch-heads/3964@{#1}
  └── Official Build:   1

FIREFOX 70 RELEASE 32-bit
├── Path:    C:\Program Files (x86)\Mozilla Firefox\firefox.exe
├── Version: 70.0.1
└─┬ Info
  ├── FileVersion:      70.0.1
  ├── LegalCopyright:   ©Firefox and Mozilla Developers; available under the MPL 2 license.
  ├── CompanyName:      Mozilla Corporation
  ├── FileDescription:  Firefox
  ├── ProductVersion:   70.0.1
  ├── InternalName:     Firefox
  ├── LegalTrademarks:  Firefox is a Trademark of The Mozilla Foundation.
  ├── OriginalFilename: firefox.exe
  ├── ProductName:      Firefox
  └── BuildID:          20191030021342

IE 11 64-bit
├── Path:    C:\Program Files\Internet Explorer\iexplore.exe
├── Version: 11.00.18362.1
└─┬ Info
  ├── FileVersion:      11.00.18362.1
  ├── CompanyName:      Microsoft Corporation
  ├── FileDescription:  Internet Explorer
  ├── InternalName:     iexplore
  ├── LegalCopyright:   © Microsoft Corporation. All rights reserved.
  ├── OriginalFilename: IEXPLORE.EXE
  ├── ProductName:      Internet Explorer
  └── ProductVersion:   11.00.18362.1

IE 11 32-bit
├── Path:    C:\Program Files (x86)\Internet Explorer\iexplore.exe
├── Version: 11.00.18362.1
└─┬ Info
  ├── FileVersion:      11.00.18362.1
  ├── CompanyName:      Microsoft Corporation
  ├── FileDescription:  Internet Explorer
  ├── InternalName:     iexplore
  ├── LegalCopyright:   © Microsoft Corporation. All rights reserved.
  ├── OriginalFilename: IEXPLORE.EXE
  ├── ProductName:      Internet Explorer
  └── ProductVersion:   11.00.18362.1

Found 7 browsers in 25 ways within 116ms.
```

That said, while this is an improvement, I haven't reproduce the reported issues, so I can't confirm that the ChromeHTML key is the main culprit.

I did manage to hit issues on this test environment due to the default 'stable' assumption from `find-update-clients` though. That's definitely incorrect for a few cases here. My installed dev & beta versions for example have exactly the same `ap` value, but different channels. Notably my stable install does explicitly say stable here (`x64-stable-statsdef_1`), so I've changed this to an explicit check.

I'm a bit suspicious of the [bitness check](https://github.com/vweevers/win-detect-browsers/blob/master/lib/chrome/find-update-clients.js#L48) in `find-update-clients` too, since it defaults to 32, which would cause [this reported issue](https://github.com/httptoolkit/feedback/issues/46#issuecomment-544325458). I can't reproduce a similar error myself here, but how would you feel about something like `ap.has('x64') ? 64 : null` instead? I'm assuming there's no explicit 32 bit value we can check for.

### CODE REVIEW

**Comment by vweevers** [comment]
> I'm a bit suspicious of the bitness check in find-update-clients too, since it defaults to 32, which would cause this reported issue. I can't reproduce a similar error myself here, but how would you feel about something like ap.has('x64') ? 64 : null instead? I'm assuming there's no explicit 32 bit value we can check for.

Let's remove that line (L48) altogether. We already have the `arch` property anyway, for all browsers.

File: lib/browsers.js [hunk 1]
```diff
@@ -6,13 +6,17 @@ exports.chrome = {
...
+    this.dir('LOCALAPPDATA', 'Google\\Chrome Dev\\Application')
     // Chrome Canary
     this.dir('LOCALAPPDATA', 'Google\\Chrome SxS\\Application')
 
     // programFiles() is a shortcut to dir() that checks both
     // "Program Files" and "Program Files (x86)" if on 64-bit Windows
     this.programFiles('Google\\Chrome\\Application')
+    this.programFiles('Google\\Chrome Beta\\Application')
+    this.programFiles('Google\\Chrome Dev\\Application')
+    this.programFiles('Google\\Chrome SxS\\Application')
```
Comment (vweevers) [inline lib/browsers.js]: Is it still the case that the offline Google Chrome installer (aka the standalone installer) installs to program files, while the normal installer installs to local app data?

File: lib/chrome/find-progids.js [hunk 2]
```diff
@@ -29,7 +29,11 @@ function queryHive (self, hive, done) {
         if (err) return next(err.notFound ? null : err)
 
         const metadata = {
-          channel: id === 'ChromeHTML' ? 'stable' : 'canary'
+          channel: id.startsWith('ChromeSSHTM.') ? 'canary' : {
+            'ChromeHTML': null, // In some cases this can be ambiguous
+            'ChromeBHTML': 'beta',
+            'ChromeDHTML': 'dev'
+          }[id]
```
Comment (vweevers) [inline lib/chrome/find-progids.js]: Can you move this logic to a function? Along the lines of:

```js
function channel (id) {
  if (id.startsWith('ChromeSSHTM.')) return 'canary'
  if (id === 'ChromeBHTML') return 'beta'
  // etc
}
```

File: lib/chrome/find-update-clients.js [hunk 3]
```diff
@@ -91,5 +94,5 @@ function getReleaseChannel (ap) {
     if (ap.has(channel)) return channel
   }
 
-  return 'stable'
+  return null
```
Comment (vweevers) [inline lib/chrome/find-update-clients.js]: IIRC the ap never contains the word `stable`, has that changed?

Comment (vweevers) [reply lib/chrome/find-update-clients.js]: Seems so; on my machine it's `x64-stable-statsdef_1`.

File: lib/browsers.js (hunk 1 above)
Comment (vweevers) [reply lib/browsers.js]: Nope, I just tried Chrome Dev, it installs to program files too. Oh well.

File: lib/chrome/find-progids.js (hunk 2 above)
Comment (pimterry) [reply lib/chrome/find-progids.js]: Done :+1:

File: lib/browsers.js (hunk 1 above)
Comment (pimterry) [reply lib/browsers.js]: It seems inconsistent, for some reason my Chrome Canary has been installed to local app data, but none of the others. No idea if I used a different installer by accident or if that's specific to canary, but it seems sensible to just check both imo.

**Comment by pimterry** [comment]
I've merge the change to drop bitness, and we're all good now I think.

Updated debug output for reference:

```
$ node cli.js --debug
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Google\Update /v LastInstallerSuccessLaunchCmdLine +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ HKCU\Software\Google\Update /v LastInstallerSuccessLaunchCmdLine +9ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe +1ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ HKCU\Software\Microsoft\Windows\CurrentVersion\App Paths\chrome.exe +1ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Clients\StartMenuInternet\Google Chrome\shell\open\command +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ HKLM\Software\Clients\StartMenuInternet\Google Chrome\shell\open\command +1ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
  win-detect-browsers   @ Software\Wow6432Node\Google\Update\ClientState\{401C381F-E0DE-4B85-8BD8-3F3F14FBDA57}\LastInstallerSuccessLaunchCmdLine +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ Software\Wow6432Node\Google\Update\ClientState\{8237E44A-0054-442C-B6B6-EA0509993955}\LastInstallerSuccessLaunchCmdLine +2ms
  win-detect-browsers Found chrome (stable):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ Software\Wow6432Node\Google\Update\ClientState\{8A69D345-D564-463C-AFF1-A69D9E530F96}\LastInstallerSuccessLaunchCmdLine +0ms
  win-detect-browsers Found chrome (canary):
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ Software\Google\Update\ClientState\{4EA16AC7-FD5A-47C3-875B-DBF4A2008C20}\LastInstallerSuccessLaunchCmdLine +1ms
  win-detect-browsers firefox: found key "70.0.1 (x86 en-US)" in HKLM\Software\Wow6432Node\Mozilla\Mozilla Firefox /v CurrentVersion +2ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files\Internet Explorer\iexplore.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Clients\StartMenuInternet\iexplore.exe\shell\open\command +1ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files\Internet Explorer\iexplore.exe
  win-detect-browsers   @ HKLM\Software\Clients\StartMenuInternet\iexplore.exe\shell\open\command +2ms
  win-detect-browsers Found chrome (canary):
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeSSHTM.BC355NTUYZBE365DOMSF45YR2M\shell\open\command +1ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeHTML\shell\open\command +1ms
  win-detect-browsers Found chrome (dev):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeDHTML\shell\open\command +0ms
  win-detect-browsers Found chrome (beta):
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ Software\Classes\ChromeBHTML\shell\open\command +1ms
  win-detect-browsers Found firefox:
  win-detect-browsers   - C:\Program Files (x86)\Mozilla Firefox\firefox.exe
  win-detect-browsers   @ HKLM\Software\Wow6432Node\Mozilla\Mozilla Firefox\70.0.1 (x86 en-US)\Main /v PathToExe +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
  win-detect-browsers   @ default location +2ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
  win-detect-browsers   @ default location +1ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
  win-detect-browsers   @ default location +0ms
  win-detect-browsers Found chrome:
  win-detect-browsers   - C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
  win-detect-browsers   @ default location +1ms
  win-detect-browsers Found firefox:
  win-detect-browsers   - C:\Program Files (x86)\Mozilla Firefox\firefox.exe
  win-detect-browsers   @ default location +2ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files (x86)\Internet Explorer\iexplore.exe
  win-detect-browsers   @ default location +25ms
  win-detect-browsers Found ie:
  win-detect-browsers   - C:\Program Files\Internet Explorer\iexplore.exe
  win-detect-browsers   @ default location +116ms
CHROME 79 BETA 64-bit
├── Path:    C:\Program Files (x86)\Google\Chrome Beta\Application\chrome.exe
├── Version: 79.0.3945.36
├── GUID:    8237E44A-0054-442C-B6B6-EA0509993955
├─┬ Info
│ ├── FileVersion:      79.0.3945.36
│ ├── CompanyName:      Google LLC
│ ├── FileDescription:  Google Chrome
│ ├── InternalName:     chrome_exe
│ ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
│ ├── OriginalFilename: chrome.exe
│ ├── ProductName:      Google Chrome
│ ├── ProductVersion:   79.0.3945.36
│ ├── CompanyShortName: Google
│ ├── ProductShortName: Chrome
│ ├── LastChange:       3582db32b33893869b8c1339e8f4d9ed1816f143-refs/branch-heads/3945@{#614}
│ └── Official Build:   1
└─┬ Uninstall
  ├── path: C:\Program Files (x86)\Google\Chrome Beta\Application\79.0.3945.36\Installer\setup.exe
  └─┬ arguments
    ├── --uninstall
    ├── --chrome-beta
    ├── --system-level
    └── --verbose-logging

CHROME 80 CANARY 64-bit
├── Path:    C:\Users\User\AppData\Local\Google\Chrome SxS\Application\chrome.exe
├── Version: 80.0.3970.0
├── GUID:    4EA16AC7-FD5A-47C3-875B-DBF4A2008C20
├─┬ Info
│ ├── FileVersion:      80.0.3970.0
│ ├── CompanyName:      Google LLC
│ ├── FileDescription:  Google Chrome
│ ├── InternalName:     chrome_exe
│ ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
│ ├── OriginalFilename: chrome.exe
│ ├── ProductName:      Google Chrome
│ ├── ProductVersion:   80.0.3970.0
│ ├── CompanyShortName: Google
│ ├── ProductShortName: Chrome
│ ├── LastChange:       2658500788f963ca730ae1050841392fedee5c91-refs/branch-heads/3970@{#1}
│ └── Official Build:   1
└─┬ Uninstall
  ├── path: C:\Users\User\AppData\Local\Google\Chrome SxS\Application\80.0.3970.0\Installer\setup.exe
  └─┬ arguments
    ├── --uninstall
    └── --chrome-sxs

CHROME 78 STABLE 64-bit
├── Path:    C:\Program Files (x86)\Google\Chrome\Application\chrome.exe
├── Version: 78.0.3904.97
├── GUID:    8A69D345-D564-463C-AFF1-A69D9E530F96
├─┬ Info
│ ├── FileVersion:      78.0.3904.97
│ ├── CompanyName:      Google LLC
│ ├── FileDescription:  Google Chrome
│ ├── InternalName:     chrome_exe
│ ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
│ ├── OriginalFilename: chrome.exe
│ ├── ProductName:      Google Chrome
│ ├── ProductVersion:   78.0.3904.97
│ ├── CompanyShortName: Google
│ ├── ProductShortName: Chrome
│ ├── LastChange:       021b9028c246d820be17a10e5b393ee90f41375e-refs/branch-heads/3904@{#859}
│ └── Official Build:   1
└─┬ Uninstall
  ├── path: C:\Program Files (x86)\Google\Chrome\Application\78.0.3904.97\Installer\setup.exe
  └─┬ arguments
    ├── --uninstall
    ├── --system-level
    └── --verbose-logging

CHROME 80 DEV 64-bit
├── Path:    C:\Program Files (x86)\Google\Chrome Dev\Application\chrome.exe
├── Version: 80.0.3964.0
├── GUID:    401C381F-E0DE-4B85-8BD8-3F3F14FBDA57
├─┬ Uninstall
│ ├── path: C:\Program Files (x86)\Google\Chrome Dev\Application\80.0.3964.0\Installer\setup.exe
│ └─┬ arguments
│   ├── --uninstall
│   ├── --chrome-dev
│   ├── --system-level
│   └── --verbose-logging
└─┬ Info
  ├── FileVersion:      80.0.3964.0
  ├── CompanyName:      Google LLC
  ├── FileDescription:  Google Chrome
  ├── InternalName:     chrome_exe
  ├── LegalCopyright:   Copyright 2019 Google LLC. All rights reserved.
  ├── OriginalFilename: chrome.exe
  ├── ProductName:      Google Chrome
  ├── ProductVersion:   80.0.3964.0
  ├── CompanyShortName: Google
  ├── ProductShortName: Chrome
  ├── LastChange:       75f9d4c9f31c6fa9302434e99ced85dc5d8e2069-refs/branch-heads/3964@{#1}
  └── Official Build:   1

FIREFOX 70 RELEASE 32-bit
├── Path:    C:\Program Files (x86)\Mozilla Firefox\firefox.exe
├── Version: 70.0.1
└─┬ Info
  ├── FileVersion:      70.0.1
  ├── LegalCopyright:   ©Firefox and Mozilla Developers; available under the MPL 2 license.
  ├── CompanyName:      Mozilla Corporation
  ├── FileDescription:  Firefox
  ├── ProductVersion:   70.0.1
  ├── InternalName:     Firefox
  ├── LegalTrademarks:  Firefox is a Trademark of The Mozilla Foundation.
  ├── OriginalFilename: firefox.exe
  ├── ProductName:      Firefox
  └── BuildID:          20191030021342

IE 11 64-bit
├── Path:    C:\Program Files\Internet Explorer\iexplore.exe
├── Version: 11.00.18362.1
└─┬ Info
  ├── FileVersion:      11.00.18362.1
  ├── CompanyName:      Microsoft Corporation
  ├── FileDescription:  Internet Explorer
  ├── InternalName:     iexplore
  ├── LegalCopyright:   © Microsoft Corporation. All rights reserved.
  ├── OriginalFilename: IEXPLORE.EXE
  ├── ProductName:      Internet Explorer
  └── ProductVersion:   11.00.18362.1

IE 11 32-bit
├── Path:    C:\Program Files (x86)\Internet Explorer\iexplore.exe
├── Version: 11.00.18362.1
└─┬ Info
  ├── FileVersion:      11.00.18362.1
  ├── CompanyName:      Microsoft Corporation
  ├── FileDescription:  Internet Explorer
  ├── InternalName:     iexplore
  ├── LegalCopyright:   © Microsoft Corporation. All rights reserved.
  ├── OriginalFilename: IEXPLORE.EXE
  ├── ProductName:      Internet Explorer
  └── ProductVersion:   11.00.18362.1

Found 7 browsers in 25 ways within 365ms.
```

**Comment by vweevers** [comment]
Released 5.0.0. Thanks!

**Comment by pimterry** [comment]
Amazing, thank you! :+1:

**Pull request merged**